WIT_Toxicity_Text_Model_Comparison.ipynb

```diff
@@ -49,7 +51,7 @@
         "# If running in colab then pip install, otherwise no need.\n",
         "try:\n",
         "  import google.colab\n",
-        "  !pip install --upgrade witwidget\n",
+        "  !pip install --upgrade tensorflow>=2.0.0 witwidget \n",
```

Comment:
do we still need this or does colab come with tf pre-installed?

Reply:
this ensures that even if someone runs this colab on a tf1 kernel, it installs tf2 and so doesn't fail. with a tf2 kernel this is a no-op

Reply:
just ensures that we use tf2 even in tf1 colab kernel, as this notebook relies on tf2

Reply:
ensures tf2 is used even in tf1 kernels